fastlane/lanes/build.rb

```diff
@@ -28,27 +75,12 @@
 
   next unless create_release
 
-  archive_zip_path = File.join(File.dirname(Dir.pwd), 'Simplenote.xarchive.zip')
-  zip(path: lane_context[SharedValues::XCODEBUILD_ARCHIVE], output_path: archive_zip_path)
-
-  version = beta_release ? ios_get_build_version : ios_get_app_version(public_version_xcconfig_file: VERSION_FILE_PATH)
+  version = beta_release ? build_code_current : release_version_current
   create_release(
     repository: GITHUB_REPO,
     version: version,
     release_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'Simplenote', 'Resources', 'release_notes.txt'),
-    release_assets: archive_zip_path.to_s,
+    release_assets: XCARCHIVE_ZIP_PATH.to_s,
     prerelease: beta_release
   )
 end
-
-lane :build_for_app_store_connect do |fetch_code_signing: true|
-  appstore_code_signing if fetch_code_signing
-
-  gym(
-    scheme: 'Simplenote',
-    workspace: WORKSPACE,
-    configuration: 'Distribution AppStore',
-    clean: true,
-    export_method: 'app-store'
-  )
-end
```

Comment:
This looks deleted but you'll find a few lines above with some additional parameters.